src/api/public/apidocs-new/paths/source_project_name_pubkey.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  summary: Get GPG key
+  description: |
+    Get the project's GPG key. If the project doesn't have a default GPG key, it looks for the first one
+    available in the namespace hierarchy, ending at the global buildservice key.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: gpgkey
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/unknown_project.yaml'
```

Comment:
I will add here the case where the pubkey is not found.

This is what I have teststed:
```
> curl -u enavarro_suse https://api-test.opensuse.org/source/home:enavarro_suse/_pubkey
Enter host password for user 'enavarro_suse':
<status code="404">
  <summary>home:enavarro_suse: no pubkey available</summary>
  <details>404 home:enavarro_suse: no pubkey available</details>
</status>
```

Adding this last case should not imply removing the already documented case of unknown project.